### PR TITLE
[Devops][SIMS-#1332]Upgrade qa environment

### PR DESCRIPTION
**Pull request by guru-aot**
- [x] Use prefix qa- where needed
 - [x] Create new Web and SIMSAPI container
 - [x] Create new DB for the SIMSAPI to connect to
 - [x] Create github action to deploy to qa
 
The QA environment uses different postgres DB (QASIMSDB) but for now uses the same test camunda(bpm) and formsflow(web). The student assessment will not work properly as the workflow is triggered in test camunda engine. Other functionalities, related to our web and api works good. 

Github actions code is checked in, the testing will be done once the code is merged.
 
 https://qa-aest-sims.apps.silver.devops.gov.bc.ca/swagger#/
 https://qa-aest-sims.apps.silver.devops.gov.bc.ca/student
https://qa-aest-sims.apps.silver.devops.gov.bc.ca/institution
https://qa-aest-sims.apps.silver.devops.gov.bc.ca/aest

### CODE REVIEW

**Review by henryoforeh-aot — APPROVED** [review]

File: .github/workflows/deploytestqa.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+name: Deploy to TEST in OpenShift
```
Comment (andrewsignori-aot) [inline .github/workflows/deploytestqa.yml:1]: Please adjust the name also.

File: .github/workflows/deploytestqa.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        default: "main"
+      BUILD_ID:
+        description: "buildid"
+        required: true
+        default: "NULL"
+
+jobs:
+  # Deploy API/App images in Test
+  deployTest:
+    name: Deploy App, API, and DB to TEST environment
```
Comment (andrewsignori-aot) [inline .github/workflows/deploytestqa.yml:18]: Please adjust the message.

File: devops/Makefile [hunk 3]
```diff
@@ -34,10 +34,50 @@ define API
...
+
+define API_PORT
+3030
+endef
+
+define DATABASE_NAME_KEY
+"database-name"
+endef
+
+define QA_API
```
Comment (andrewsignori-aot) [inline devops/Makefile:53]: The declaration of the specific QA variables does not seem to be following the DEV/TEST deployment.
Can we not handle the variables in .env file for local, or git hub actions?

**Review by andrewsignori-aot — COMMENTED** [review]
Please take a look at the comments.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the changes, looks good 👍

**Review by dheepak-aot — APPROVED** [review]
LGTM @guru-aot 👍

**Pull request merged**